Core/Workflow/Step.py

```diff
@@ -44,7 +43,7 @@ def __init__( self, step_type = None, obj = None, parent = None ):
       self.parameters = ParameterCollection( self, obj.parameters )
       self.module_instances = InstancesPool( self, obj.module_instances )
       if obj.module_definitions != None:
```

Comment:
This library has been written so long ago, in a quite "strange" manner, and it is so central, that I don't really want to touch it much. Not in this PR, at least.